src/main/scala/com/fluidcode/configuration/Configuration.scala

```diff
@@ -63,6 +66,14 @@ case class Configuration(
     createTable(spark, emptyConf.toDF(), tableProperties, partitionColumns = null, overwrite)
   }
 
+  def initPostInfoTable(spark: SparkSession, overwrite: Boolean = false): Boolean = {
+    import spark.implicits._
+    val location = s"$rootPath/$database/$postInfoTable"
+    val tableProperties = TableProperties(database, postInfoTable, location)
+    val emptyConf: Seq[Silver] = Seq()
```

Comment:
naming the case class Silver is too broad. we have multiple tables in the silver layer. Pick a specific name for the case class.